backend/services/exceptions.py

```diff
@@ -10,3 +16,16 @@ def __init__(self, action: str, resource: str):
         super().__init__(
             f'Not authorized to perform `{action}` on `{resource}`')
 
+class EventNotFoundException(Exception):
+    """EventNotFoundException is raised when trying to access an event that does not exist."""
+
+    def __init__(self, id: int | None):
+        super().__init__(
+            f'No event found with matching ID: {id}')
+
+class OrganizationNotFoundException(Exception):
+    """OrganizationNotFoundException is raised when trying to access an organization that does not exist."""
+
+    def __init__(self, id: str):
+        super().__init__(
+            f'No organization found matching slug/id: {id}')
```

Comment:
I would be inclined to do one of two things:

1. Simplify by simply using `ResourceNotFoundException` anywhere we're using these more specific types and remove these types entirely.
2. Change the superclass of these two types to be `ResourceNotFoundException`.

The motivation for either is because we have a top-level exception handler in FastAPI for responding 404 when a `ResourceNotFoundException` is encountered.